packages/runtime/rollup.config.js

```diff
@@ -1,24 +1,59 @@
-const path = require('path');
-const alias = require('@rollup/plugin-alias');
 const replace = require('@rollup/plugin-replace');
 const copy = require('rollup-plugin-copy');
+const semver = require('semver');
 
 const FEDERATION_DEBUG = process.env.FEDERATION_DEBUG || '';
 
 module.exports = (rollupConfig, projectOptions) => {
   rollupConfig.input = {
     index: 'packages/runtime/src/index.ts',
-    type: 'packages/runtime/src/type/index.ts',
+    types: 'packages/runtime/src/types.ts',
     helpers: 'packages/runtime/src/helpers.ts',
   };
 
   const project = projectOptions.project;
   const pkg = require(project);
+  const RELEASE_NUMBER = Number(
+    `${semver.major(pkg.version)}${semver.minor(pkg.version)}${semver.patch(
+      pkg.version,
+    )}`,
+  );
+
+  if (Number.isNaN(RELEASE_NUMBER)) {
+    throw new Error(
+      `RELEASE_NUMBER is not valid number , please check current pkg.version is valid.`,
+    );
+  }
+  if (rollupConfig.output.format === 'esm' && FEDERATION_DEBUG) {
+    rollupConfig.output.format = 'iife';
+    rollupConfig.output.inlineDynamicImports = true;
+    delete rollupConfig.external;
+    delete rollupConfig.input.type;
+    delete rollupConfig.input.helpers;
+  }
+
+  if (rollupConfig.output.format === 'esm' && FEDERATION_DEBUG) {
+    rollupConfig.output.format = 'iife';
+    rollupConfig.output.inlineDynamicImports = true;
+    delete rollupConfig.external;
+    delete rollupConfig.input.type;
+    delete rollupConfig.input.helpers;
+  }
 
   rollupConfig.plugins.push(
+    // alias({
+    //   entries: [
+    //     {
+    //       find: '@module-federation/sdk',
+    //       replacement: path.resolve(__dirname, '../../dist/packages/sdk'),
+    //     },
+    //   ],
+    // }),
     replace({
+      preventAssignment: true,
       __VERSION__: `'${pkg.version}'`,
```

Comment:
Why does the version need a single quotation mark
![image](https://github.com/module-federation/universe/assets/27547179/37084896-ea96-4198-9986-82632443b1d3)

Reply:
not need , let me remove

Reply:
![image](https://github.com/module-federation/universe/assets/41466093/c2e1d9b3-66e8-4690-a625-44ac52e5781f)

Reply:
![image](https://github.com/module-federation/universe/assets/41466093/83d17579-bb62-44f5-8b79-d74c1f52ff77)

![image](https://github.com/module-federation/universe/assets/41466093/d08acf84-738d-4875-a653-d74713715e5b)